build_automation/build_automation/settings.py

```diff
@@ -122,8 +122,16 @@
 # https://docs.djangoproject.com/en/2.0/howto/static-files/
 
 STATIC_URL = '/static/'
+STATIC_ROOT = ''
```

Comment:
If we're not going to use this, can't we safely delete this line?

Reply:
I need to look at this.